### DISCUSSION TITLE
Add support for php 7.4

### DISCUSSION THREAD

**Discussion by realtebo** [spatie/laravel-activitylog#900] [Ideas] [answered]
We have a monster project that actually is using Laravel 8 but with php 7.4

We're interested in using this package, but it requires php 8.0

**Comment by Gummibeer** [comment, marked as answer]
Hey,
I'm sorry but we won't downgrade to PHP7.4 again.
You can still use v3 of the activitylog package. It's stable and works with all your requirements. 
https://github.com/spatie/laravel-activitylog/tree/v3
https://spatie.be/docs/laravel-activitylog/v3/introduction

**Reply by realtebo** [reply]
Ok, thanks. I'll check v3
